src/app/security-group/sg-rules/sg-rules.component.html

```diff
@@ -109,7 +109,7 @@
                   matInput
                   name="icmpCode"
                   maxValue="255"
-                  [minValue]="cidrIpVersion === IPVersion.ipv6 ? 0 : -1"
+                  minValue="0"
```

Comment:
It should be possible to use ICMP code -1 for IPv4 rules. The same for ICMP types.

Reply:
Please provide link that ICMP Type can have "-1" value. I can't find that info on the internet.
https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml